src/UI/Art/SpriteArtist/SpriteInfoView.java

```diff
@@ -54,12 +53,17 @@ public void render(Graphics g)
   {
     DiagonalBlindsBG.draw(g);
 
-    // Get the CO info menu.
+    // Get the draw space. We'll draw it all in real size and then scale it when we draw to the window.
     int drawScale = SpriteOptions.getDrawScale();
+    Dimension dimensions = SpriteOptions.getScreenDimensions();
+    int imageWidth = dimensions.width / drawScale;
+    int imageHeight = dimensions.height / drawScale;
+    BufferedImage image = SpriteLibrary.createTransparentSprite(imageWidth, imageHeight);
+    Graphics myG = image.getGraphics();
     
-    int paneOuterBuffer = 4*drawScale; // sets both outer border and frame border size
-    int paneHSize = (int) (viewWidth*0.7) - paneOuterBuffer*4; // width of the BG color area
-    int paneVSize = (int) (viewHeight   ) - paneOuterBuffer*4; // height ""
```

Comment:
May as well get rid of the `viewWidth` and height variables, since they're no longer used.

Reply:
`getPreferredDimensions()`. Unless you want to have that just invoke `SpriteOptions.getScreenDimensions()`

Reply:
Actually, no reason not to just do that.